ingest/lambdas/rss-feed-ingester.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable import/no-unresolved, import/extensions */
+const pMap = require('p-map');
+
+const dspaceClient = require('../../lib/dspace-client');
+const ingestQueue = require('../lib/ingest-queue');
+const { getIntFromEnv, getStringFromEnv } = require('../../lib/env-utils');
+
+/**
+ * Returns a boolean indicating whether or not we should publish documents
+ * from the RSS feed based on when the RSS feed was published and the
+ * last time we checked it. Set the `feedReadInterval` to -1 to always
+ * pass this test.
+ *
+ * @param {Date} publishedDate The published date of the RSS feed.
+ * @param {number} feedReadInterval The number in seconds since the
+ * published RSS feed was last read.
+ *
+ * @returns {boolean} Whether or not we should publish new documents based
+ *          on the last time we checked the RSS feed
+ */
+const shouldQueuePublishedDocuments = (publishedDate, feedReadInterval) => {
+  if (feedReadInterval === -1) {
+    return true;
+  }
+
+  // Assuming the last time we checked for an updated feed was feedReadInterval
+  // seconds ago, if the published date is newer than that we return true.
+  const feedReadIntervalDate = Date.now() - feedReadInterval;
```

Comment:
`Operator '>=' cannot be applied to types 'Date' and 'number'.ts(2365)`

Reply:
Fixed (I think) but why don't I see that compiler error?